modules/trusted-firmware-m/tfm_boards/nrf54l15_cpuapp/config.cmake

```diff
@@ -13,3 +13,11 @@ include(${PLATFORM_PATH}/common/${NRF_SOC_VARIANT}/config.cmake)
 
 # Override PS_CRYPTO_KDF_ALG
 set(PS_CRYPTO_KDF_ALG                  PSA_ALG_SP800_108_COUNTER_CMAC CACHE STRING    "KDF Algorithm to use")
+
+
+# attest_hal.c includes bl_storage.h, which needs CONFIG_NRFX_RRAMC to be defined.
+# This is because bl_stoarge is a lib intended to be run from either the bootloader (Zephyr) or from
+# TF-M. CONFIG_NRFX_RRAMC can also not be set for NSPE, so we can not inherit this from app Kconfig.
```

Comment:
Maybe clearer like this?
```suggestion
# TF-M. This is independent from the NS image's CONFIG_NRFX_RRAMC, which can be disabled, so we can not inherit this from app Kconfig.
```

Reply:
RRAMC is not even available from the NSPE image, so how about this?
```suggestion
# TF-M. This is independent from the NS image's CONFIG_NRFX_RRAMC, which must be disabled, so we can not inherit this from app Kconfig.
```

Reply:
good to me